SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -99,17 +113,38 @@ public final class SwiftGenStoryboardEnumBuilder {
             
             text += "\(s)\(t)}\n"
         }
-        if enumName != nil {
-            text += "\(t)}\n"
+        text += "\(t)}\n"
+      
+        /// Segues
+        if (storyboardsSegues.count > 0) {
+          text += "\n"
+          
+          text += "\(t)enum \(seguesEnumName.asSwiftIdentifier()) {\n"
+        
+          for (name, segues) in storyboardsSegues
+            where segues.count > 0 {
+              
+            let enumName = name.asSwiftIdentifier(forbiddenChars: "_")
+            text += "\(s)\(t)enum \(enumName) : String {"
+            
+            for segue in segues {
+              let caseName = segue.segueID.asSwiftIdentifier(forbiddenChars: "_")
+              text += "\n\(s)\(t)\(t)/// \(caseName)\n"
+              text += "\(s)\(t)\(t)case \(caseName)Segue = \"\(caseName)\"\n"
```

Comment:
it is fact __*not*__ necessary.

It actually reads wrong. I'm going to take it out as well as the `= "\(caseName)\"` assignment since 2.0 doesn't require that for `String` cases whose `rawValue` is not provided and is inferred to be the same as the `case` which IMHO reads better.